examples/creatives/simid_overlay.js

```diff
@@ -39,4 +40,21 @@ class SimidOverlay extends BaseSimidCreative {
     document.getElementById(elementName).addEventListener(
         'click', sendMessageFunction.bind(this));
   }
+
+    /**
+   * Listens for a click event on the log button and sends the
+   *   message contained in the input field to the player.
+   * @private
+   */
```

Comment:
nit, fix the indentation on the first line. The stars (the first star of the first line and then all the other stars) should be aligned

Reply:
fixed